core/src/main/java/com/kodebeagle/javaparser/MethodInvocationResolver.java

```diff
@@ -544,4 +590,84 @@ public String toString() {
                     '}';
         }
     }
+
+    public static class JavaDoc {
+
+        private String name;
+        private String comment;
+
+        public String getName() {
+            return name;
+        }
+
+        public void setName(String name) {
+            this.name = name;
+        }
+
+        public String getComment() {
+            return comment;
+        }
+
+        public void setComment(String comment) {
+            this.comment = comment;
+        }
+
+        public JavaDoc(String name, String comment) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)